### ISSUE TITLE
Issue about details of the model

### ISSUE THREAD

**Issue by YangGao-hash** [HowieMa/PPT#13]
Hello, I have reproduced both Transfusion and the PPT (and have correctly processed the data). I found that the results from my training are different each time, and I am unable to achieve the accuracy of the model you provided. Could the issue be due to only using 2D supervision？I have a few images with extremely inaccurate results：

    "s_09_act_10_subact_01_000647": 178.32215821244446,
    "s_09_act_10_subact_01_000711": 214.48203716800427,

    "s_11_act_10_subact_02_000364": 420.0389947248091,
    "s_11_act_10_subact_02_000428": 329.85906022658537,

and these are yours：
    "s_09_act_10_subact_01_000647": 64.20743666358514,
    "s_09_act_10_subact_01_000711": 89.07862228210108,

    "s_11_act_10_subact_02_000364": 53.7054512754271,
    "s_11_act_10_subact_02_000428": 106.18530511204406,

How can I fix this question？Just train for many times and pick a good result?

**Comment by HowieMa** [comment]
Hi, thanks for your interests in our work! 

I only trained it once and get these results, not "pick a good with many times". 

As for your results, there could be many reasons. 
1) I am not sure if you have exact the same environment as mine. Over the past two years, Cuda, Pytorch, numpy, etc have been updated frequently.
2) If you can get very good results on the training, while just bad results on the eval datasets. I guess you may not initialize the transformers from pre-trained PPT-S on 256x256 (https://github.com/HowieMa/PPT/blob/main/multi-view-PPT/README.md?plain=1#L17). I also found that if you just random initialize the transformer from scratch and train it on Human 3.6M only, you may easily overfit the training sets and fail on the eval sets.
